05.ls/ls.rb

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'etc'
+
+class Command
+  # def command_exec(option)
+  #   ls_command(**option)
+  # end
+
+  def command_exec(a: nil, l: nil, r: nil)
+    files_name = if a
+                   Dir.glob('*', File::FNM_DOTMATCH)
+                 else
+                   Dir.glob('*')
+                 end
+    files_name.reverse! if r
+
+    if l
+      l_opt(files_name)
+    else
+      none_opt(files_name)
+    end
+  end
+
+  private
+
+  def none_opt(files_name)
+    array = []
+    files_num = files_name.length
+    row_num = (files_num / 3.to_f).ceil
+
+    row_num.times do |_i|
```

Comment:
いらないならブロック引数書かなくても大丈夫です 👍

```suggestion
    row_num.times do
```

Reply:
あと、配列の初期化がしたいだけであればわざわざループしなくてもこんな感じで書けます 👍 （どちらでも大丈夫です！）

```ruby
array = Array.new(row_num) { [] }
```

なお、下記だと全て同じオブジェクト(`[]`)が詰められてしまってうまく動かないのでご注意ください〜

```ruby
array = Array.new(row_num, [])
```

Reply:
承知いたしました！